portal-api/src/v1/api.js

```diff
@@ -47,6 +53,11 @@ const createDeal = async (deal, user) => {
 
 const updateDeal = async (dealId, dealUpdate, user) => {
   try {
+    if (!isValidMongoId(dealId)) {
+      console.error('api.updateDeal - invalid dealId provided ', dealId);
```

Comment:
Please can we have consistent console.error as follows:

```javascript
console.error('Update deal API failed for deal id %s', dealId);
```

Same applies to all of the below please.